ledger/sample/demo.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"github.com/tron-us/go-btfs-common/ledger"
+	"log"
+
+	ic "github.com/libp2p/go-libp2p-core/crypto"
+)
+
+const (
+	PayerPrivKeyString    = "CAISIJFNZZd5ZSvi9OlJP/mz/vvUobvlrr2//QN4DzX/EShP"
+	ReceiverPrivKeyString = "CAISIDm/qF5f98Jh8FGBUcFUhQvJPU8uEah1SZrR1BrGekC0"
+)
+
+func main() {
+	ctx := context.Background()
+	// build connection with ledger
+	clientConn, err := ledger.LedgerConnection("ledger-dev.bt.co:443", "")
+	if err != nil {
+		log.Panic("fail to connect", err)
+	}
+	defer ledger.CloseConnection(clientConn)
+	// new ledger client
+	ledgerClient := ledger.NewClient(clientConn)
+	// create payer Account
+	payerPrivKey, err := convertToPrivKey(PayerPrivKeyString)
+	if err != nil {
+		log.Panic(err)
+	}
+	payerPubKey := payerPrivKey.GetPublic()
+	//_, err = ledger.ImportAccount(ctx, payerPubKey, ledgerClient)
+	_, err = ledger.ImportSignedAccount(ctx, payerPrivKey, payerPubKey, ledgerClient)
+	if err != nil {
+		log.Panic(err)
+	}
+	// create receiver account
+	recvPrivKey, err := convertToPrivKey(ReceiverPrivKeyString)
+	if err != nil {
+		log.Panic(err)
+	}
+	recvPubKey := recvPrivKey.GetPublic()
+	//_, err = ledger.ImportAccount(ctx, recvPubKey, ledgerClient)
+	_, err = ledger.ImportSignedAccount(ctx, recvPrivKey, recvPubKey, ledgerClient)
+	if err != nil {
+		log.Panic(err)
+	}
+	// prepare channel commit
+	amount := int64(1)
+	channelCommit, err := ledger.NewChannelCommit(payerPubKey, recvPubKey, amount)
+	if err != nil {
+		log.Panic(err)
+	}
+	// sign for the channel commit
+	fromSig, err := ledger.Sign(payerPrivKey, channelCommit)
```

Comment:
Seems Sign() and Verify() has been moved to https://github.com/TRON-US/go-btfs-common/blob/master/crypto/crypto.go. Can you check this can be run? Thanks.

Reply:
fixed